egs/librispeech/asr/simple_v1/mmi_att_transformer_decode.py

```diff
@@ -294,22 +288,11 @@ def main():
     HLG.requires_grad_(False)
 
     # load dataset
-    feature_dir = Path('exp/data')
+    librispeech = LibriSpeechAsrDataModule(args)
     test_sets = ['test-clean', 'test-other']
-    for test_set in test_sets:
+    for test_set, test_dl in zip(test_sets, librispeech.train_dataloaders()):
```

Comment:
Should it be `librispeech.test_dataloaders()`?

Reply:
Thanks, fixed. I'll re-run the whole thing to make sure it works alright.